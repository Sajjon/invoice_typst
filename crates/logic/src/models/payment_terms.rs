use crate::prelude::*;

/// The payment terms of this invoice, e.g. `Net { due_in: 30 }`
#[derive(Clone, Debug, Serialize, Deserialize)]
pub enum PaymentTerms {
    /// Net payment due in a specific number of days, e.g. `Net(30)`
    Net(u8),
}

impl PaymentTerms {
    pub fn net30() -> Self {
        PaymentTerms::Net(30)
    }
}
