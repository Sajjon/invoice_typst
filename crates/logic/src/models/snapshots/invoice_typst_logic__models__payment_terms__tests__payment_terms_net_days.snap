---
source: crates/logic/src/models/payment_terms.rs
expression: net_days
---
"Net 30"
